examples_tests/postgres_test.go

```diff
@@ -19,8 +19,6 @@ func TestPostgresTestSuite(t *testing.T) {
 }
 
 func (s *PostgresTestSuite) TestPostgres() {
-	s.T().Parallel()
-
```

Comment:
Why did you remove parallel runs?

Reply:
This was spotted by the linter. I believe that the warning corresponds to [this](https://github.com/stretchr/testify/issues/187) issue, which is 10 years old and doesn't look like it wouldd be resolved in `V1`, so the safer option is to avoid parallel run for test suits.